client/src/components/UI/Modal/Modal.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Modal.css';
+
+const modal = (props) => {
+  return (
+    <div className="Modal">
+      {props.children}
+    </div>
+  )
+}
+
+export default modal;
```

Comment:
@simonnguyen3054 Then when you export the component you should capitalize it here as well to 'Modal'.